templates/artist/includes/artist_detail_overview.html

```diff
@@ -46,8 +47,11 @@ <h4 class="investing">${{ campaign.amount_raised|intcomma }}</h4>
                         <button class="button login-box" data-open="login-modal">LOGIN</button>
                     {% endif %}
                 {% else %}
+                    <h3 class="campaign-funded">FUNDED!</h3>
```

Comment:
That's true... So would it make sense to have {% if campaign.funded %} for this?